patches/@ephox+bedrock-server+11.5.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js b/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
+index 0a33f63..c3c6926 100644
+--- a/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
++++ b/node_modules/@ephox/bedrock-server/lib/main/ts/bedrock/compiler/Webpack.js
+@@ -38,7 +38,7 @@ const webpackSharedRules = webpackRemap.concat([
+         enforce: 'pre'
+     },
+     {
+-        test: /\.(html|htm|css|bower|hex|rtf|xml|yml)$/,
++        test: /\.(html|htm|css|bower|hex|rtf|xml|yml|svg)$/i,
```

Comment:
For others reading, this was needed as Bedrock can't currently load images and for TinyMCE we're stuck on an older version of Bedrock atm so we can't just add this to bedrock and upgrade. So this was a solution Andy and I discussed that we thought was best until we can upgrade, as this shouldn't be needed elsewhere for older bedrock versions.

Reply:
Didn't know that we had all those types of imports already in bedrock.